### PR TITLE
fix: update tasks cloning to include labels

### PR DESCRIPTION
**Pull request by TCL735**
Part 2 (tasks) of idpe 13138

- Cloning tasks will now also clone the labels

https://user-images.githubusercontent.com/10736577/155248265-72f78e1b-e80f-4da6-9283-67ee47182b21.mp4

### CODE REVIEW

File: cypress/e2e/shared/tasks.test.ts [hunk 1]
```diff
@@ -211,26 +211,45 @@ from(bucket: "defbuck")
     })
 
     it('can clone a task and activate just the cloned one', () => {
-      createTask('task1', 'buckets()')
```
Comment (TCL735) [inline cypress/e2e/shared/tasks.test.ts:214]: There is already an existing task in the test suite. There is no need to create another task. We're testing labels and cloning here, not task creation.

File: cypress/e2e/shared/tasks.test.ts [hunk 2]
```diff
@@ -687,22 +706,3 @@ from(bucket: "defbuck")
...
-    .click()
-
-  cy.getByTestID('flux-editor').monacoType(task)
-
-  cy.getByTestIDAndSetInputValue('task-form-name', name)
-  cy.getByTestIDAndSetInputValue('task-form-offset-input', offset)
-  cy.getByTestIDAndSetInputValue('task-form-schedule-input', every)
-  cy.getByTestID('task-save-btn').click()
-  cy.getByTestID('notification-success--dismiss').click()
-}
```
Comment (TCL735) [inline cypress/e2e/shared/tasks.test.ts:708]: redundant ✂️

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3944)

**Review by hoorayimhelping — COMMENTED** [review]
I'll wait to hear your thoughts on my review of https://github.com/influxdata/ui/pull/3943 before reviewing this :+1:

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**